sphinx_needs/utils.py

```diff
@@ -161,6 +186,29 @@ def row_col_maker(
                 else:
                     ref_col.append(text_col)
                     para_col += ref_col
+            elif matching_link_confs:
+                try:
+                    link_name = None
```

Comment:
Hi @iSOLveIT,
Really cool Idea and implementation.
After a first look, the code here and the code in layout.py codeline 482 mostly look the same.
Is it possible to define a function once and reuse ist in both places?

Reply:
Yeah I realised that but I will have to check that out properly. Anyway, thank you for the suggestion.

Reply:
@PhilipPartsch 
I have implemented the suggestions in recent push changes.
Thanks again.